### ISSUE TITLE
One-liner description uses deprecated "should" syntax

### ISSUE THREAD

**Issue by dominicsayers** [rspec/rspec-core#2581]
### Subject of the issue
When you use `format --documentation` against a spec expressed in the one-liner syntax, the example description is formed using "should".

This is contrary to modern usage where a spec's description is encouraged to be a statement about what the subject *is* not what it *should be*.

### Your environment
* Ruby version: ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin17]
* rspec-core version: rspec-core 3.8.0

### Steps to reproduce

Use the following script as described in the [bug reporting template](https://github.com/rspec/rspec-core/blob/master/REPORT_TEMPLATE.md)

```
# frozen_string_literal: true

begin
  require "bundler/inline"
rescue LoadError => e
  $stderr.puts "Bundler version 1.10 or later is required. Please update your Bundler"
  raise e
end

gemfile(true) do
  source "https://rubygems.org"

  gem "rspec", "3.8.0" # Activate the gem and version you are reporting the issue against.
end

puts "Ruby version is: #{RUBY_VERSION}"
require 'rspec/autorun'

RSpec.describe 'one-liner description uses deprecated "should" syntax' do
  subject { 1 + 1 }

  it { is_expected.to eq(2) } # => "  should eq 2"
end
```

### Expected behavior
I expect the following output:

```
one-liner description uses deprecated "should" syntax
  is expected to eq 2
```

or even better

```
one-liner description uses deprecated "should" syntax
  is 2
```

### Actual behavior

```

```
one-liner description uses deprecated "should" syntax
  should eq 2
```

**Comment by dominicsayers** [comment]
Thanks. I *did* search the outstanding issues before I raised this one. Clearly I missed that one!

Not sure I understand your CI workflow; is it bad that you got [Travis failures](https://travis-ci.org/rspec/rspec-core/jobs/449098572) when the fix was merged to master?

Anyway, hope this can get released soon. Will there be another 3.x release before 4.0?

**Comment by JonRowe** [comment]
> Not sure I understand your CI workflow; is it bad that you got Travis failures when the fix was merged to master?

Yes if they are real failures, they weren't they got restarted and passed. You'll see an all green history.

> Anyway, hope this can get released soon. Will there be another 3.x release before 4.0?

Most certainly, RSpec 4 is a long way off.
